2.train-VAE/betatcvae.py

```diff
@@ -0,0 +1,300 @@
+# betatcvae.py
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from typing import List
+import pandas as pd 
+import numpy as np
+import pathlib
+import math
```

Comment:
Consider making these in alphabetical order using PEP8 import standards. 

Might consider using isort with pre-commit : https://github.com/WayScience/MNIST_TD-VAE/blob/master/.pre-commit-config.yaml

Reply:
Fixed